Proposed (#3683)
modelica/ModelicaSpecification: chapters/annotations.tex

\subsection{Variable Graphics and Schematic Animation}\label{variable-graphics-and-schematic-animation}

> Any value except when restricted otherwise in their respective definitions, for example with \lstinline!/* literal */ constant!) in graphical annotations can be dependent on class variables using \lstinline!DynamicSelect!.

Reviewer: ```suggestion
Any value in graphical annotations can be dependent on class variables using \lstinline!DynamicSelect! except when restricted otherwise in their respective definitions, (for example with \lstinline!/* literal */ constant!).
```

> Any value in graphical annotations can be dependent on class variables using \lstinline!DynamicSelect! except when restricted otherwise in their respective definitions, (for example with \lstinline!/* literal */ constant!).

Reviewer: ```suggestion
Any value in graphical annotations can be dependent on evaluable parameters except when restricted otherwise in their respective definitions, (for example with \lstinline!/* literal */ constant!).
```

\lstinline!DynamicSelect! has the syntax of a function call with two arguments, where the first argument specifies the value of the static state and the second argument the value of the dynamic state.

> The first follow the rule given below.

Author: ```suggestion
The first argument follow the rule given below.
```

> The first argument follow the rule given below.

Reviewer: ```suggestion
The first argument follows the rule given below.
```

> The first argument follows the rule given below.

Reviewer: The first argument follows the same rules as when not using \lstinline!DynamicSelect!.

Reviewer: This was supposed to be a "suggestion"...

```suggestion
The first argument follows the same rules as when not using \lstinline!DynamicSelect!.
```

> The second argument may contain references to variables to enable a dynamic behavior.

Reviewer: ```suggestion
The second argument may contain references to variables of a higher variability to enable displaying a dynamic behavior of a simulation.
```

> If \lstinline!DynamicSelect! is not used (or for the first argument in \lstinline!DynamicSelect!), expressions dependent on evaluable parameters are allowed, but not expressions with a higher variability.

Reviewer: ```suggestion
```
